tests/linearizability/traffic.go

```diff
@@ -17,6 +17,8 @@ package linearizability
 import (
 	"context"
 	"fmt"
+	clientv3 "go.etcd.io/etcd/client/v3"
+	"go.etcd.io/etcd/tests/v3/linearizability/model"
```

Comment:
nit: import groups

Reply:
done

Reply:
FYI: `./scripts/fix.sh` takes care of the import groups.